include/clap/entry.h

```diff
@@ -23,23 +23,36 @@ extern "C" {
 //   - /Library/Audio/Plug-Ins/CLAP
 //   - ~/Library/Audio/Plug-Ins/CLAP
 //
-// Additionally, extra path may be specified in CLAP_PATH environment variable.
-// CLAP_PATH is formated in the same way as the OS' binary search path (PATH on UNIX, Path on Windows).
+// In addition to the OS-specific default locations above, a CLAP host must query the environment
+// for a CLAP_PATH variable, which is a list of directories formatted in the same manner as the host
+// OS binary search path (PATH on Unix, separated by `:` and Path on Windows, separated by ';', as
+// of this writing).
 //
-// Every methods must be thread-safe.
+// Each directory should be recursively searched for files and/or bundles as appropriate in your OS
+// ending with the extension `.clap`.
+//
+// On Windows and Linux, the '.clap' file is a renamed shared library object, so traversing the path
+// should stop at the first non-directory (regular file, symlink, etc...) whose name ends in '.clap'.
```

Comment:
I doubt anyone will do it wrong, but this now implies that the host should stop searching when it encounters a symlink to a directory called `.clap`. The host should just always follow symlinks during the search. That way there doesn't need to be any special distinction, as the host will stop as regular files (after resolving the symlinks) with a `.clap` extension.

Reply:
Yeah good catch, symlink is dirty business with cycles and so on.

Reply:
Basically you'll just want to `realpath()` (or the equivalent thereof) every thing in your directory walker. If you use a library for walking the directories then this likely already has that built in as an option.

Reply:
I really think that 
```C
// Each directory should be recursively searched for files and/or bundles as appropriate in your OS
// ending with the extension `.clap`.
```
says everything.